File: bindings/ruby/smoke_test.rb
$LOAD_PATH.unshift(File.expand_path("lib", __dir__))
require "gurmukhi/gurmukhi"

result = Gurmukhi.to_unicode("gurU", Gurmukhi::UnicodeStandard::SANT_LIPI)
raise "FAIL: expected 'ਗੁਰੂ', got '#{result}'" unless result == "ਗੁਰੂ"
puts "PASS: Ruby"

File: bindings/python/smoke_test.py
import sys
import os

sys.path.insert(0, os.path.dirname(__file__))

from gurmukhi.gurmukhi import to_unicode, UnicodeStandard

result = to_unicode("gurU", UnicodeStandard.SANT_LIPI)
assert result == "ਗੁਰੂ", f'FAIL: expected "ਗੁਰੂ", got "{result}"'
print("PASS: Python")
